README.md

```diff
@@ -4,17 +4,49 @@
 
 This is my to-do list website built with html, css and JavaScript. In this project I used Webpack to bundle JavaScript and ES6 modules to write modular JavaScript. 
 
+![screenshot](./Screenshot.png)
+
+## Built With
+
+- Webpack
+- HTML
+- CSS
+- Modular JavaScript (with modern ES6 syntax)
+
+## Live Demo
+
+[Live Demo Link](https://mwapsam.github.io/To-Do-List/)
+
 ## Getting Started
 
 **Setup**
 
 git clone https://github.com/Mwapsam/To-Do-List.git
 
+Execute the following commands on your terminal
 ```
 npm install
 npm start
 ```
 
+### Prerequisites
+
+You need to have NODE.JS set up on your local machine.
+
+
+### Deployment
+
+Execute the following commands on your terminal
+
+```
+npm run build
+npm run deploy
+```
+
+It will open your browser with the project homepage
+
+
+
 ## Authors
```

Comment:
- [ ] OPTIONAL
Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.